sdk/communication/azure-communication-rooms/src/main/java/module-info.java

```diff
@@ -2,17 +2,18 @@
 // Licensed under the MIT License.
 
 module com.azure.communication.rooms {
+    requires com.azure.json;
     requires transitive com.azure.communication.common;
 
     // public API surface area
     exports com.azure.communication.rooms;
     exports com.azure.communication.rooms.models;
 
     opens com.azure.communication.rooms
-        to com.fasterxml.jackson.databind, com.azure.core;
+        to com.azure.core;
```

Comment:
If I recall correctly, the guide says to also add this to `module-info.java`.
```suggestion
        to com.azure.core, com.azure.json;
```

Reply:
Nope, we don't need to specify `com.azure.json` specifically (same as the pom should only need com.azure).
From the [wiki](https://github.com/Azure/azure-sdk-for-java/wiki/Azure-Json-Migration): 
![image](https://github.com/Azure/azure-sdk-for-java/assets/16845631/a215f966-6ce3-48a4-809a-9dab038534f3)